From 85e1afe9eb1a174b1cb4b72a0aa1eb2d99e7c3ad Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sat, 11 Aug 2018 14:48:52 +0300
Subject: [PATCH] hl: make: fix dali-distclean

Only remove the HL_VARS_FILE after the projects have been cleaned.
Otherwise, we get:

boot/uboot/uboot.mk:347: *** No board defconfig name specified, check your BR2_TARGET_UBOOT_BOARD_DEFCONFIG setting.  Stop.
make[2]: Leaving directory '/home/igrinberg/trees/npu-stack/buildroot'
Makefile:41: recipe for target 'uboot-dirclean' failed
make[1]: *** [uboot-dirclean] Error 2
make[1]: Leaving directory '/home/igrinberg/trees/npu-stack/buildroot-external'
Makefile:150: recipe for target 'dali-distclean' failed
make: *** [dali-distclean] Error 2

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 0681a7d..91be1bd 100644
--- a/Makefile
+++ b/Makefile
@@ -147,11 +147,11 @@
 gaudi-distclean: hl-gaudi
 dali-distclean: hl-dali
 dali-distclean gaudi-distclean: $(OUTPUT_DIR)/.config
-	@rm -f $(HL_VARS_FILE)
 	@$(MAKE) uboot-dirclean linux-dirclean armcp-dirclean	\
 		 spdlog-dirclean specs-dirclean specs_external-dirclean \
 		 host-uboot-test-hooks-dirclean host-tbot-dirclean \
 		 thermal-dirclean preboot-dirclean host-fw_loader-dirclean
+	@rm -f $(HL_VARS_FILE)
 
 .PHONY: dali-onix-test dali-pldm-test dali-asic-test
 dali-onix-test: dali-onix-tools-install
